### DISCUSSION TITLE
How can I contribute to updating the Documentation?

### DISCUSSION THREAD

**Discussion by dave-wilson-tech** [Megabit/Blazorise#4966] [General]
I found some errors in the documentation at https://blazorise.com/docs/components/layout
How can I contribute to improving this?
Thanks guys, totally in love with Blazorise!

**Comment by stsrki** [comment]
Hello @dave-wilson-tech. Thank you for your interest in your contribution. Any help is appreciated 😊

General guidelines can be found at https://github.com/Megabit/Blazorise/blob/master/CONTRIBUTING.md

You can target the `rel-1.3` branch as that is our current release branch, and is going to be visible online soon after we approve any changes. 

Thanks once again.
